Documentation/ReleaseNotes.md

```diff
@@ -145,7 +145,25 @@ The audio spatializer can now be specified in the MRTK configuration dialog. Ins
 
 ### Breaking changes in 2.4.0
 
-**Eye gaze setup change**
+**Eye Gaze API**
+
+The `UseEyeTracking` property from `GazeProvider` implementation of `IMixedRealityEyeGazeProvider`  was renamed to `IsEyeTrackingEnabled`.
+
+If you did this previously...
+
+```csharp
+GazeProvider gazeProvider = FindObjectOfType<GazeProvider>();
+gazeProvider.UseEyeTracking = true;
```

Comment:
Do we want to recommend `CoreServices.InputSystem.GazeProvider` instead of `FindObjectsByType`? That way we have a single static reference to it instead of searching the scene.

```suggestion
if (CoreServices.InputSystem.GazeProvider is GazeProvider gazeProvider)
{
    gazeProvider.UseEyeTracking = true;
}
```